### ISSUE TITLE
Support managing a Generic folder

### ISSUE THREAD

**Issue by Al12rs** [Nexus-Mods/NexusMods.App#1646]
## Problem:
We currently only support Stardew Valley. 
Developers and enthusiast users want to try the app for other games.
Linux and mac users don't have many manager alternatives, but want to play other games.

## Proposal:
Allow managing a generic folder. 
Together with Advanced installer it is possible to install any mod inside a folder.
No Nexus Download support, since there is no game, but you can install from archives on disk.

## Impact:
- Improved dogfooding (since devs can use app for whatever game they wish)
- More opportunity for user feedback

### Implementation details:
As discussed in meeting:
- CLI command to manage a folder
- Add a Generic Game similar to stubbed game, with most things stubbed out.
- Consider adding a button in ui to manage a folder, likely hidden behind experimental setting

**Comment by Pickysaurus** [comment]
I don't think this is a good idea at this point. We should stay focused on the limited scope of games we are supporting. But once the app is a bit more mature this may be useful.

**Comment by Al12rs** [comment]
The main problem this is trying to solve is the fact that while we are still limited in scope, we don't get to test the core features as much as we would like, because we don't personally play Stardew or xyz game in our free time.

**Comment by LukeNexusMods** [comment]
Which core features are you unable to test? And why?

**Comment by Idun9** [comment]
Would be nice to have this feature added. Of course it will be fully unsupported but at least grouping games and mods running directly on OS instead of running wine/lutris launcher would be very enjoyable.

**Comment by EzioTheDeadPoet** [comment]
I like the idea of this especially for testing it out.
